overlay.d/05rhcos/usr/lib/dracut/modules.d/30rhcos-afterburn-checkin/module-setup.sh

```diff
@@ -19,5 +19,7 @@ check() {
 install() {
     local unit=rhcos-afterburn-checkin.service
     inst_simple "$moddir/$unit" "$systemdsystemunitdir/$unit"
-    systemctl -q --root="$initdir" add-requires ignition-files.service "$unit"
+    # note we `|| exit 1` here so we error out if e.g. the units are missing
+    # see https://github.com/coreos/fedora-coreos-config/issues/799
+    systemctl -q --root="$initdir" add-requires ignition-files.service "$unit" || exit 1
```

Comment:
I guess this dracut module should technically be re-ordered to 31 at least so it's more obvious that it's after `30ignition`. Though the fact that we have `|| exit 1` here clearly demonstrates that it works fine.